packages/admin-ui/src/components/Sidebar/components/SidebarBackdrop.tsx

```diff
@@ -47,52 +52,58 @@ export const SidebarBackdrop = forwardRef(function SidebarBackdrop(
           bg: '$secondary',
           borderRight: selectedItem?.expandable ? '$neutral' : 'none',
         }}
-        onMouseEnter={showToggle}
-        onMouseLeave={hideToggle}
       >
-        {loading && <SidebarItemSkeleton />}
+        {showSidebarItemSkeleton && <SidebarItemSkeleton />}
       </tag.div>
 
-      <Button
-        title="toggle sidebar collapse"
-        name="toggle sidebar collapse"
-        variant="tertiary"
-        csx={{
-          left:
-            selectedItem?.expandable && reduced
-              ? distance.reduced
-              : distance.expanded,
-          opacity: selectedItem?.expandable && toggleVisible ? 1 : 0,
-          position: 'absolute',
-          zIndex: 1,
-          top: '5rem',
-          cursor: 'pointer',
-          borderRadius: '100%',
-          border: '$neutral',
-          height: '1.5rem',
-          width: '1.5rem',
-          transition:
-            'left 200ms cubic-bezier(0.4, 0.14, 0.3, 1), opacity 175ms cubic-bezier(0.4, 0.14, 0.3, 1)',
-          bg: '$primary',
-        }}
-        icon={
-          <IconCaretRight
-            mirrored={!reduced}
-            height="0.875rem"
-            width="0.875rem"
+      {!loading ? (
```

Comment:
Not a fan of a "not this variable value". 
Any chance we could `loading ? "loading state" : "not loading state"`?

Reply:
Or maybe a `const wantedState = !loading` and `wantedState && "this"`

Reply:
Got it, I can change this. It will like that:
```jsx
{loading ? null : <Component />}
```
What do you think?